idle-rpg/game/utils/Event.js

```diff
@@ -284,6 +296,20 @@ class Event {
   // Item Events
   generateTownItemEvent(discordHook, twitchBot, selectedPlayer) {
     return new Promise((resolve) => {
+      const horseChance = Helper.randomBetween(1, 5);
+      if(horseChance === 5){
+        const horseCost = 200;
+        if (selectedPlayer.gold >= horseCost){
+          selectedPlayer.horse = 'Yes';
+        }
```

Comment:
Should verify if player already has a horse or not, also have to remove the horsecost from players gold